spark/dl/src/test/scala/com/intel/analytics/bigdl/utils/tf/TensorflowLoaderSpec.scala

```diff
@@ -483,6 +483,98 @@ class TensorflowLoaderSpec extends TensorflowSpecHelper{
     }
   }
 
+  "TensorArray operations" should "be load correctly" in {
+    val output = Seq("scatter_and_gather:0", "split_and_concat:0", "write_and_read:0", "size1:0",
+      "size2:0", "unstack_and_stack:0")
+    val comparePairs = testModel("tensor_array", output, backward = false)
+    for (i <- output.indices) {
+      val (tf, bigdl) = comparePairs(i)
+      tf.almostEqual(bigdl, 1e-6) should be(true)
+    }
+    for (i <- output.length until comparePairs.length) {
```

Comment:
If backward is false, then this for loop is  not needed.

Reply:
All the following test cases

Reply:
fixed